src/connectors/panoptic/panoptic.controllers.ts

```diff
@@ -808,24 +929,26 @@ export async function getAccountLiquidity(
   ethereumish: Ethereumish,
   panopticish: Panoptic,
   req: GetAccountLiquidityRequest
-): Promise<string> {
+): Promise<GetAccountLiquidityResponse> {
   const { wallet } = await txWriteData(ethereumish, req.address);
-  const liquidity = await panopticish.getAccountLiquidity(
+  const result = await panopticish.getAccountLiquidity(
     wallet,
     req.univ3pool,
     req.owner,
     req.tokenType,
     req.tickLower,
     req.tickUpper
   );
-  return liquidity;
+  return {
+    accountLiquidities: result.accountLiquidities
```

Comment:
ABI suggests it's just one number, though also uses "accountLiquidities" instead of "accountLiquidity". I went ahead and renamed it: 8238fd11f6f781a5d14cfd78e3133122dbdd4d50